Ladybird/LocationEdit.cpp

```diff
@@ -39,6 +39,9 @@ void LocationEdit::highlight_location()
 {
     auto url = AK::URL::create_with_url_or_path(ak_deprecated_string_from_qstring(text()));
 
+    auto placeholder_text_color = QPalette().color(QPalette::Text);
```

Comment:
Since we're not using this for a "placeholder" per se, seems like we can allow ourselves to call it something better :)